package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "revai-node-sdk",
-  "version": "3.2.0",
+  "version": "3.3.0",
```

Comment:
If imports have potentially been changed this needs a major version bump

Reply:
I don't think any existing imports have been changed